src/core/a-animation.js

```diff
@@ -363,6 +364,8 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
     getForCoordinateComponent();
   } else if (['true', 'false'].indexOf(dataTo) !== -1) {
     getForBoolean();
+  } else if (dataTo && dataTo[0] === '#') {
```

Comment:
Alright I'm about to do my hour drive home and then I'll try to parse through the schema code to make these adjustments.  May not get it done until tomorrow.  Thanks for all the feedback. @dmarcos @ngokevin